# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1745610053 -7200
# Parent  5b6062791b26d29ec53e7c554775ab4882889760
Fix regression from bug 1900211 when OWL is used.

diff --git a/mailnews/compose/src/MessageSend.sys.mjs b/mailnews/compose/src/MessageSend.sys.mjs
--- a/mailnews/compose/src/MessageSend.sys.mjs
+++ b/mailnews/compose/src/MessageSend.sys.mjs
@@ -1257,15 +1257,15 @@ export class MessageSend {
       false
     );
 
     for (const addr of addresses) {
-      let displayName = addr.name;
+      let displayName = addr.name || "";
       // If we know this is a list, or it seems likely, don't collect the
       // displayName which may contain the sender's name instead of the (only)
       // name of the list.
       if (
         this._compType == Ci.nsIMsgCompType.ReplyToList ||
-        addr.name.includes(" via ")
+        displayName.includes(" via ")
       ) {
         displayName = "";
       }
       lazy.collectSingleAddress(addr.email, displayName, createCard);
